examples/guides/se-restful-webservice/README.adoc

```diff
@@ -70,135 +73,78 @@ See the <<downloading,downloading>> section for instructions.
+Maven project for a Helidon service.
+--
+
+=== Browse the generated source
+
+==== `pom.xml`
+The input you provided to the archetype determines the project's Maven
+coordinates:
+[source,xml,indent=0]
+// _include::0-0:{pom}[tag=coordinates]
+----
+    <artifactId>se-restful-webservice</artifactId>
+----
 
-=== Update your `pom.xml`
-Add the properties and dependencies listed here if they don't exist in your `pom.xml`:
+Also in the `<dependency-management>` section notice the dependency on the Helidon bom pom:
 [source,xml,subs="verbatim,attributes"]
-// _include::1-3:{pom}[tags=helidonVersion;javaVersions]
-// _include::8-14:{pom}[tag=depMgt]
-----
-    <properties>
-        <helidon.version>0.11.1-SNAPSHOT</helidon.version>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
-    </properties>
-
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>io.helidon</groupId>
-                <artifactId>helidon-bom</artifactId>
-                <version>${helidon.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-----
-Now you can add Helidon dependencies to your project without having to specify the version.
-For this example add these dependencies:
-[source,xml]
-// _include::1-8:{pom}[tags=webserverBundleDependency;configYamlDependency]
-----
-    <dependencies>
-        <dependency> <!--1-->
-            <groupId>io.helidon.bundles</groupId>
-            <artifactId>helidon-bundles-webserver</artifactId>
-        </dependency>
-        <dependency> <!--2-->
-            <groupId>io.helidon.config</groupId>
-            <artifactId>helidon-config-yaml</artifactId>
-        </dependency>
-    </dependencies>
-----
-<1> Incorporates the Helidon web server.
-<2> Pulls in support for YAML as a config format.
-
-If you run your project from the IDE, the IDE typically handles the main class and places
-dependent JARs on the runtime classpath for you and your pom is now ready to go.
-
-// tag::runMavenOutsideIDE[]
-==== To run Maven outside your IDE
-If you want to use Maven yourself,
-outside the IDE, then add the following to your pom. (This is typical with Maven
-projects and is not specific to Helidon or this example):
+// _include::0-6:{pom}[tag=bom,indent=0]
+----
+<dependency>
+    <groupId>io.helidon</groupId>
+    <artifactId>helidon-bom</artifactId>
+    <version>${helidon.version}</version>
+    <type>pom</type>
+    <scope>import</scope>
+</dependency>
+----
+
+Lower, in the `<dependencies>` section, you will see declarations for various
+Helidon components:
+
+* web server
+* config support for YAML
+* health check
+* metrics
+
+The bom adds `<dependency-management>` declarations for all the Helidon components.
```

Comment:
The 'helidon-bom' dependency adds `<dependency-management>` declarations for all the Helidon components.

Reply:
Fixed in PR #364 which addresses changes to the SE tutorial.